src/org/zaproxy/zap/extension/globalexcludeurl/DialogAddToken.java

```diff
@@ -117,6 +123,18 @@ protected void init() {
         getEnabledCheckBox().setSelected(true);
         getDescTextField().setText("");
         token = null;
+        
+		// Handle escape key to close the dialog
+		KeyStroke escape = KeyStroke.getKeyStroke(KeyEvent.VK_ESCAPE, 0, false);
```

Comment:
The `getFieldsPane` suggestion was only if this was going to be applied to (just) `DialogAddToken` and `DialogModifyToken`. Sorry for not being clear.